Grails CLI 6.2.1 does not create Java 11 compatible apps

```console
mattias@mattias-xps:~/test$ java -version
openjdk version "11.0.17" 2022-10-18
OpenJDK Runtime Environment Temurin-11.0.17+8 (build 11.0.17+8)
OpenJDK 64-Bit Server VM Temurin-11.0.17+8 (build 11.0.17+8, mixed mode)
mattias@mattias-xps:~/test$ grails -V
Grails Version: 6.2.1
JVM Version: 17.0.9
mattias@mattias-xps:~/test$ grails create-app g621app
| Application created at /home/mattias/test/g621app
mattias@mattias-xps:~/test$ cd g621app/
mattias@mattias-xps:~/test/g621app$ ./gradlew bootRun
Starting a Gradle Daemon, 3 incompatible and 1 stopped Daemons could not be reused, use --status for details

> Task :bootRun FAILED
Error: LinkageError occurred while loading main class g621app.Application
        java.lang.UnsupportedClassVersionError: g621app/Application has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':bootRun'.
> Process 'command '/home/mattias/.sdkman/candidates/java/11.0.17-tem/bin/java'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 22s
6 actionable tasks: 6 executed
```

--- a/.github/workflows/mavenCentral.yml
+++ b/.github/workflows/mavenCentral.yml
@@ -18,7 +18,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
-          java-version: '17'
+          java-version: '11'
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
       - name: "🔐 Generate secring file"

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -21,7 +21,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
-          java-version: '17'
+          java-version: '11'
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
       - name: "🔢 Set the current release version"
@@ -128,7 +128,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
-          java-version: '17'
+          java-version: '11'
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
       - name: "✅ Run Tests"
@@ -186,7 +186,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
-          java-version: '17'
+          java-version: '11'
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
       - name: "✅ Run Tests"
@@ -235,7 +235,7 @@ jobs:
       - name: "☕️ Setup GraalVM CE"
         uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '17'
+          java-version: '11'
           distribution: 'graalvm-community'
           native-image-musl: 'true'
           components: 'native-image'
@@ -280,7 +280,7 @@ jobs:
       - name: "☕️ Setup GraalVM CE"
         uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '17'
+          java-version: '11'
           distribution: 'graalvm-community'
           components: 'native-image'
           github-token: ${{ secrets.GITHUB_TOKEN }}
@@ -324,7 +324,7 @@ jobs:
       - name: "☕️ Setup GraalVM CE"
         uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '17'
+          java-version: '11'
           distribution: 'graalvm-community'
           components: 'native-image'
           github-token: ${{ secrets.GITHUB_TOKEN }}
@@ -368,7 +368,7 @@ jobs:
       - name: "☕️ Setup GraalVM CE"
         uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '17'
+          java-version: '11'
           distribution: 'graalvm-community'
           components: 'native-image'
           github-token: ${{ secrets.GITHUB_TOKEN }}
@@ -419,7 +419,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
-          java-version: '17'
+          java-version: '11'
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
       - name: "🚀 Grails SDK Minor Release"

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -236,9 +236,9 @@ jobs:
         uses: graalvm/setup-graalvm@v1
         with:
           java-version: '11'
-          distribution: 'graalvm-community'
           native-image-musl: 'true'
           components: 'native-image'
+          version: '22.3.3'
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
@@ -281,8 +281,8 @@ jobs:
         uses: graalvm/setup-graalvm@v1
         with:
           java-version: '11'
-          distribution: 'graalvm-community'
           components: 'native-image'
+          version: '22.3.3'
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
@@ -325,8 +325,8 @@ jobs:
         uses: graalvm/setup-graalvm@v1
         with:
           java-version: '11'
-          distribution: 'graalvm-community'
           components: 'native-image'
+          version: '22.3.3'
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
@@ -369,8 +369,8 @@ jobs:
         uses: graalvm/setup-graalvm@v1
         with:
           java-version: '11'
-          distribution: 'graalvm-community'
           components: 'native-image'
+          version: '22.3.3'
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4

--- a/.github/workflows/sdkman.yml
+++ b/.github/workflows/sdkman.yml
@@ -19,7 +19,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
-          java-version: '17'
+          java-version: '11'
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
       - name: "🚀 Grails SDK Minor Release"

--- a/.github/workflows/snapshot.yml
+++ b/.github/workflows/snapshot.yml
@@ -156,8 +156,8 @@ jobs:
         uses: graalvm/setup-graalvm@v1
         with:
           java-version: '11'
-          distribution: 'graalvm-community'
           components: 'native-image'
+          version: '22.3.3'
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
@@ -192,8 +192,8 @@ jobs:
         uses: graalvm/setup-graalvm@v1
         with:
           java-version: '11'
-          distribution: 'graalvm-community'
           components: 'native-image'
+          version: '22.3.3'
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
@@ -230,8 +230,8 @@ jobs:
         uses: graalvm/setup-graalvm@v1
         with:
           java-version: '11'
-          distribution: 'graalvm-community'
           components: 'native-image'
+          version: '22.3.3'
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
@@ -266,8 +266,8 @@ jobs:
         uses: graalvm/setup-graalvm@v1
         with:
           java-version: '11'
-          distribution: 'graalvm-community'
           components: 'native-image'
+          version: '22.3.3'
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4

--- a/.github/workflows/snapshot.yml
+++ b/.github/workflows/snapshot.yml
@@ -218,6 +218,7 @@ jobs:
           name: grails-darwin-amd64-snapshot
           path: grails-darwin-amd64-snapshot.zip
   macos-arm:
+    if: false # Graalvm cannot build Java 11 OS X Arm Native Image
     name: "Build OS X Arm Native CLI"
     runs-on: macos-latest
     env:

--- a/.github/workflows/snapshot.yml
+++ b/.github/workflows/snapshot.yml
@@ -17,7 +17,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
-          java-version: '17'
+          java-version: '11'
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
       - name: "🔨 Run Build"
@@ -62,7 +62,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
-          java-version: '17'
+          java-version: '11'
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
       - name: "✅ Run Tests"
@@ -118,7 +118,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'temurin'
-          java-version: '17'
+          java-version: '11'
       - name: "🐘 Setup Gradle"
         uses: gradle/actions/setup-gradle@v4
       - name: "✅ Run Tests"
@@ -155,7 +155,7 @@ jobs:
       - name: "☕️ Setup GraalVM CE"
         uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '17'
+          java-version: '11'
           distribution: 'graalvm-community'
           components: 'native-image'
           github-token: ${{ secrets.GITHUB_TOKEN }}
@@ -191,7 +191,7 @@ jobs:
       - name: "☕️ Setup GraalVM CE"
         uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '17'
+          java-version: '11'
           distribution: 'graalvm-community'
           components: 'native-image'
           github-token: ${{ secrets.GITHUB_TOKEN }}
@@ -229,7 +229,7 @@ jobs:
       - name: "☕️ Setup GraalVM CE"
         uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '17'
+          java-version: '11'
           distribution: 'graalvm-community'
           components: 'native-image'
           github-token: ${{ secrets.GITHUB_TOKEN }}
@@ -265,7 +265,7 @@ jobs:
       - name: "☕️ Setup GraalVM CE"
         uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '17'
+          java-version: '11'
           distribution: 'graalvm-community'
           components: 'native-image'
           github-token: ${{ secrets.GITHUB_TOKEN }}

